service-base/src/test/java/org/eclipse/hono/service/tenant/CompleteBaseTenantServiceTest.java

```diff
@@ -37,15 +35,16 @@
 import org.junit.rules.Timeout;
 import org.junit.runner.RunWith;
 
-import io.opentracing.Span;
-import io.vertx.core.AsyncResult;
-import io.vertx.core.Future;
-import io.vertx.core.Handler;
-import io.vertx.core.json.JsonArray;
-import io.vertx.core.json.JsonObject;
-import io.vertx.core.net.SelfSignedCertificate;
-import io.vertx.ext.unit.TestContext;
-import io.vertx.ext.unit.junit.VertxUnitRunner;
+import javax.security.auth.x500.X500Principal;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+import java.security.cert.X509Certificate;
+import java.util.Optional;
+import java.util.UUID;
```

Comment:
this is the wrong import order, please use the order defined in `eclipse/hono.importorder`

Reply:
fixed.